src/kakarot/instructions/environmental_information.cairo

```diff
@@ -521,8 +531,15 @@ namespace EnvironmentalInformation {
         // Update context stack.
         let ctx = ExecutionContext.update_stack(self=ctx, new_stack=stack);
         // Increment gas used.
-        // TODO: compute gas (incidentally need to discern whether address is cold)
-        let ctx = ExecutionContext.increment_gas_used(self=ctx, inc_value=GAS_COST_EXTCODECOPY);
+        let (minimum_word_size) = Helpers.compute_minimum_word_for_size_of_minimum_word_count(
```

Comment:
`compute_minimum_word_for_size_of_minimum_word_count` I don't really understand what this name means. The goal of this helper is to compute the number of words (ie number of `bytes32`) required to store `size` bytes, hence `(size + 31) / 32` (EVM works with slots of 32 bytes, so you require only one slot for storing 1 to 32 bytes, 2 slots for 33 to 64, etc.)